{"name": "Негативный тест логина: несколько ошибок", "status": "failed", "statusDetails": {"message": "AssertionError: assert \"{'username':...вторизации']}\" == 'Incorrect us...я авторизации'\n  + {'username': ['Incorrect username length'], 'password': ['Необходимо указать пароль для авторизации']}\n  - Incorrect username length\n  - Необходимо указать пароль для авторизации", "trace": "self = <test_ui.TestUI object at 0x10a9916d0>, user = '123456789000000000'\n\n    @screenshot_on_failure\n    @allure.story('Логин')\n    @allure.title('Негативный тест логина: несколько ошибок')\n    @pytest.mark.parametrize('user', ['1234', '123456789000000000'])\n    def test_login_negative_multiple_errors(self, user):\n        \"\"\"\n        Тестируется форма логина.\n        Происходит авторизация пользователя с несколькими ошибками (длина и пустой пароль)\n        Шаги:\n        1. Вводятся данные пользователя с паролем с пробелами и неккоректной длиной имени пользователя.\n        2. Нажимается кнопка логина.\n        Ожидается, что появятся все сообщения о полученных ошибках в читаемом формате.\n        Проверяется наличие сообщения об ошибке.\n        \"\"\"\n        self.login_page.login(user, '   ')\n    \n        elem = self.login_page.find(self.login_page.locators.LOGIN_ERROR)\n        while elem.text == '':\n            elem = self.login_page.find(self.login_page.locators.LOGIN_ERROR)\n    \n>       assert elem.text == \"Incorrect username length\\nНеобходимо указать пароль для авторизации\"\nE       assert \"{'username':...вторизации']}\" == 'Incorrect us...я авторизации'\nE         + {'username': ['Incorrect username length'], 'password': ['Необходимо указать пароль для авторизации']}\nE         - Incorrect username length\nE         - Необходимо указать пароль для авторизации\n\nelem       = <selenium.webdriver.remote.webelement.WebElement (session=\"597b40b58ef1e132895253f40af937ff\", element=\"16ede73e-d06d-4d92-8fae-fd8da814942d\")>\nself       = <test_ui.TestUI object at 0x10a9916d0>\nuser       = '123456789000000000'\n\ntests/test_ui.py:168: AssertionError"}, "description": "\n        Тестируется форма логина.\n        Происходит авторизация пользователя с несколькими ошибками (длина и пустой пароль)\n        Шаги:\n        1. Вводятся данные пользователя с паролем с пробелами и неккоректной длиной имени пользователя.\n        2. Нажимается кнопка логина.\n        Ожидается, что появятся все сообщения о полученных ошибках в читаемом формате.\n        Проверяется наличие сообщения об ошибке.\n        ", "attachments": [{"name": "Screenshot", "source": "92883e44-af5f-4a5b-a6ed-5c7c21b59ba2-attachment.png", "type": "image/png"}], "parameters": [{"name": "user", "value": "'123456789000000000'"}], "start": 1590591730892, "stop": 1590591732956, "uuid": "a4d8c852-3e91-44b4-b637-204a4db57258", "historyId": "fbfa8944d6c7430e59d6b703b1988c80", "testCaseId": "efc4d42af754dd91a940b9241449ee46", "fullName": "tests.test_ui.TestUI#test_login_negative_multiple_errors", "labels": [{"name": "feature", "value": "UI"}, {"name": "story", "value": "Логин"}, {"name": "tag", "value": "UI"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "TestUI"}, {"name": "host", "value": "MacBook-Pro-Lev.local"}, {"name": "thread", "value": "46107-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}